### DISCUSSION TITLE
Have you thought to add Bluetooth at command?

### DISCUSSION THREAD

**Discussion by nxor** [JAndrassy/WiFiEspAT#106] [Ideas]
Would be great to also be able to use Bluetooth over AT command in esp32...

Any thoughts on this?

**Comment by JAndrassy** [comment]
yes I thought about BLE (not classic BT) it and decided that I will not do it becuase
1) There are better BLE options than esp32. for example nRF5 MCUs. 
2) The AT 2 firmware is not good so I focus on esp8266 AT 1 and Jiri Bilek's fw.

If I would do BLE, I would implement an adapter for the ArduinoBLE library.
